rosdep/python.yaml

```diff
@@ -3721,13 +3721,7 @@ python-wstool:
   fedora: [python-wstool]
   gentoo: [dev-python/wstool]
   macports: [py27-wstool]
-  ubuntu:
-    precise: [python-wstool]
-    quantal: [python-wstool]
-    raring: [python-wstool]
-    saucy: [python-wstool]
-    trusty: [python-wstool]
-    trusty_python3: [python-wstool]
+  ubuntu: [python-wstool]
```

Comment:
unfortunately collapsing them will prevent users from using the python3 version of the rule.
Though trusty_python3 should be `python3-wstool` and not `python-wstool`.
@tfoote I see this comes from #4018. Do you prefer to fix the python3 rule or collapse them ?